apps/sertra/pages/create.tsx

```diff
@@ -56,6 +57,10 @@ const SubmitServer: NextPage<{ swrFallback: { [key: string]: unknown } }> = ({
 
   return (
     <SWRConfig value={{ fallback: swrFallback }}>
+      <HeadWrapper
+        title="Submit Server"
```

Comment:
You've changed the names of the props but haven't updated this and another place where the component is used.